src/rating-menu/rating-menu.ts

```diff
@@ -59,7 +59,7 @@ export type RatingMenuState = {
           >
             <svg
               *ngFor="let star of item.stars"
-              [ngClass]="cx('starIcon')"
+              [ngClass]="cx('starIcon') + ' ' + (star ? cx('starIcon', 'full') : cx('starIcon', 'empty'))"
```

Comment:
might be slighly shorter `cx('starIcon', star ? 'full' : 'empty')`

Reply:
Right, it always have either `full` or `empty` 👍

Reply:
Oh no, if I do that, it will generate only `ais-RatingMenu-starIcon--xxx` without `ais-RatingMenu-starIcon`. `cx` function doesn't do that.